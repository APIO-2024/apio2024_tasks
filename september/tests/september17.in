txy4h26c1rm1uv8tr3eonahd67u8h56x
53
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 7 8 5 6 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
8 9 7 5 6 3 2 4 1
10 1
0 1 2 3 4 5 6 7 8
8 9 6 7 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
8 9 7 5 6 3 4 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 5 4 2 3 1
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 5 3 2 1 4
10 1
0 1 2 3 4 5 6 7 8
9 7 8 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
8 9 7 6 5 4 3 1 2
10 1
0 1 2 3 4 5 6 7 8
9 8 6 7 5 3 4 1 2
10 1
0 1 2 3 4 5 6 7 8
9 7 6 8 5 3 2 1 4
10 1
0 1 2 3 4 5 6 7 8
8 7 6 9 5 4 2 3 1
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 5 4 2 3 1
10 1
0 1 2 3 4 5 6 7 8
8 7 9 6 5 4 3 1 2
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
8 9 7 6 5 3 4 2 1
10 1
0 1 2 3 4 5 6 7 8
8 9 7 5 6 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 7 6 8 4 5 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 5 4 3 1 2
10 1
0 1 2 3 4 5 6 7 8
8 9 7 6 5 4 3 1 2
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
39 38 37 35 36 34 33 32 30 31 28 29 26 25 27 24 23 22 21 20 19 18 16 17 15 14 13 11 12 10 9 7 8 6 5 4 2 3 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
39 37 38 36 35 34 33 32 31 29 30 27 28 26 25 24 23 21 22 19 18 20 17 16 15 14 13 11 12 10 8 9 7 6 5 3 2 4 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
39 38 37 35 36 34 33 32 31 29 30 28 26 27 25 23 22 21 24 20 18 19 16 15 17 14 12 11 10 13 9 7 8 6 4 5 3 1 2
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
39 38 37 35 36 34 32 33 30 29 31 27 28 25 24 26 23 21 22 19 20 18 17 15 14 16 12 11 10 9 13 7 6 5 8 3 4 2 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
38 39 37 36 34 35 33 32 31 30 28 29 26 25 27 24 22 21 20 23 19 17 16 15 18 13 12 14 11 10 9 7 6 8 4 3 5 2 1
100 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98
98 99 97 96 95 93 94 92 90 91 89 88 87 86 85 84 83 81 82 80 79 78 77 75 74 73 76 72 70 71 69 67 68 65 64 66 63 61 62 59 60 58 56 57 54 55 53 52 50 51 49 47 48 46 45 44 42 43 40 39 38 41 36 37 35 33 32 34 30 29 31 27 28 25 26 23 24 22 20 19 21 17 16 18 15 13 12 14 10 9 8 11 7 5 6 4 3 1 2
1000 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 99 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115 116 117 118 119 120 121 122 123 124 125 126 127 128 129 130 131 132 133 134 135 136 137 138 139 140 141 142 143 144 145 146 147 148 149 150 151 152 153 154 155 156 157 158 159 160 161 162 163 164 165 166 167 168 169 170 171 172 173 174 175 176 177 178 179 180 181 182 183 184 185 186 187 188 189 190 191 192 193 194 195 196 197 198 199 200 201 202 203 204 205 206 207 208 209 210 211 212 213 214 215 216 217 218 219 220 221 222 223 224 225 226 227 228 229 230 231 232 233 234 235 236 237 238 239 240 241 242 243 244 245 246 247 248 249 250 251 252 253 254 255 256 257 258 259 260 261 262 263 264 265 266 267 268 269 270 271 272 273 274 275 276 277 278 279 280 281 282 283 284 285 286 287 288 289 290 291 292 293 294 295 296 297 298 299 300 301 302 303 304 305 306 307 308 309 310 311 312 313 314 315 316 317 318 319 320 321 322 323 324 325 326 327 328 329 330 331 332 333 334 335 336 337 338 339 340 341 342 343 344 345 346 347 348 349 350 351 352 353 354 355 356 357 358 359 360 361 362 363 364 365 366 367 368 369 370 371 372 373 374 375 376 377 378 379 380 381 382 383 384 385 386 387 388 389 390 391 392 393 394 395 396 397 398 399 400 401 402 403 404 405 406 407 408 409 410 411 412 413 414 415 416 417 418 419 420 421 422 423 424 425 426 427 428 429 430 431 432 433 434 435 436 437 438 439 440 441 442 443 444 445 446 447 448 449 450 451 452 453 454 455 456 457 458 459 460 461 462 463 464 465 466 467 468 469 470 471 472 473 474 475 476 477 478 479 480 481 482 483 484 485 486 487 488 489 490 491 492 493 494 495 496 497 498 499 500 501 502 503 504 505 506 507 508 509 510 511 512 513 514 515 516 517 518 519 520 521 522 523 524 525 526 527 528 529 530 531 532 533 534 535 536 537 538 539 540 541 542 543 544 545 546 547 548 549 550 551 552 553 554 555 556 557 558 559 560 561 562 563 564 565 566 567 568 569 570 571 572 573 574 575 576 577 578 579 580 581 582 583 584 585 586 587 588 589 590 591 592 593 594 595 596 597 598 599 600 601 602 603 604 605 606 607 608 609 610 611 612 613 614 615 616 617 618 619 620 621 622 623 624 625 626 627 628 629 630 631 632 633 634 635 636 637 638 639 640 641 642 643 644 645 646 647 648 649 650 651 652 653 654 655 656 657 658 659 660 661 662 663 664 665 666 667 668 669 670 671 672 673 674 675 676 677 678 679 680 681 682 683 684 685 686 687 688 689 690 691 692 693 694 695 696 697 698 699 700 701 702 703 704 705 706 707 708 709 710 711 712 713 714 715 716 717 718 719 720 721 722 723 724 725 726 727 728 729 730 731 732 733 734 735 736 737 738 739 740 741 742 743 744 745 746 747 748 749 750 751 752 753 754 755 756 757 758 759 760 761 762 763 764 765 766 767 768 769 770 771 772 773 774 775 776 777 778 779 780 781 782 783 784 785 786 787 788 789 790 791 792 793 794 795 796 797 798 799 800 801 802 803 804 805 806 807 808 809 810 811 812 813 814 815 816 817 818 819 820 821 822 823 824 825 826 827 828 829 830 831 832 833 834 835 836 837 838 839 840 841 842 843 844 845 846 847 848 849 850 851 852 853 854 855 856 857 858 859 860 861 862 863 864 865 866 867 868 869 870 871 872 873 874 875 876 877 878 879 880 881 882 883 884 885 886 887 888 889 890 891 892 893 894 895 896 897 898 899 900 901 902 903 904 905 906 907 908 909 910 911 912 913 914 915 916 917 918 919 920 921 922 923 924 925 926 927 928 929 930 931 932 933 934 935 936 937 938 939 940 941 942 943 944 945 946 947 948 949 950 951 952 953 954 955 956 957 958 959 960 961 962 963 964 965 966 967 968 969 970 971 972 973 974 975 976 977 978 979 980 981 982 983 984 985 986 987 988 989 990 991 992 993 994 995 996 997 998
998 999 997 995 996 994 993 992 991 990 989 988 987 986 985 983 984 982 981 980 978 979 977 976 975 973 974 972 971 969 970 968 967 965 966 963 964 961 960 962 958 959 956 957 955 954 953 951 950 952 948 947 949 945 946 943 944 942 941 940 939 937 936 938 935 933 932 931 934 929 930 928 926 925 927 924 922 923 921 919 920 918 916 915 914 917 913 912 911 910 909 908 906 907 904 905 903 901 902 900 898 899 896 897 895 894 893 892 891 890 889 888 886 885 887 883 884 882 881 880 878 877 876 875 874 879 873 871 872 870 868 869 867 866 865 864 862 863 861 860 859 858 856 857 855 853 854 852 851 849 850 848 846 845 844 843 847 842 840 839 838 837 841 836 834 833 832 835 831 829 830 828 827 826 825 823 824 821 820 822 819 817 816 815 818 814 813 812 811 810 808 807 806 805 809 804 802 803 800 801 799 798 797 796 794 795 792 793 790 791 789 788 787 785 784 783 782 781 780 779 778 786 776 777 774 775 773 771 770 772 768 769 767 765 764 763 766 761 762 760 759 758 756 757 755 754 753 751 750 752 748 747 749 746 745 743 744 741 740 739 738 737 742 735 734 733 736 731 730 732 729 727 726 725 728 724 722 723 721 720 719 717 716 715 718 713 714 712 710 709 711 707 708 705 706 704 703 702 700 699 698 701 696 697 695 693 694 691 690 692 688 689 686 685 684 687 682 683 681 680 678 679 676 675 677 674 672 673 671 670 668 667 666 669 664 663 665 661 662 659 660 658 657 655 654 656 653 652 651 650 648 649 646 647 644 643 645 641 642 639 638 637 640 635 634 636 633 632 631 630 629 628 627 626 625 624 622 621 620 623 619 617 616 615 614 618 613 611 610 609 612 608 607 606 604 603 602 605 601 600 598 597 596 595 594 593 599 592 590 591 589 588 586 587 585 584 583 582 580 579 581 577 576 578 575 574 573 572 571 570 568 567 569 565 566 564 562 563 560 559 561 557 558 555 556 554 553 551 552 550 548 549 547 546 544 545 543 542 540 539 541 537 538 535 534 536 533 531 532 530 528 527 529 526 525 523 522 524 521 520 519 518 517 516 514 513 515 511 512 510 508 507 509 505 506 504 503 502 501 500 499 498 497 496 495 494 493 491 490 492 488 489 487 486 484 485 482 483 480 479 481 478 476 477 475 474 472 473 470 469 471 468 467 465 464 466 463 462 460 459 461 458 457 455 456 453 454 451 452 450 449 447 448 446 444 445 442 443 440 439 441 438 436 437 434 433 435 432 431 430 428 427 426 429 424 423 422 421 425 420 418 419 417 416 415 413 412 411 414 409 410 408 407 406 404 405 402 403 400 399 398 401 397 395 396 394 392 393 391 390 389 387 388 385 384 386 383 381 382 379 380 378 376 375 374 377 373 371 372 370 369 368 367 366 364 365 362 363 360 359 358 361 357 355 354 353 356 351 350 352 349 348 347 346 344 343 342 341 345 339 338 337 336 340 335 333 334 331 330 329 332 327 328 326 325 323 324 321 320 322 318 319 316 315 317 314 313 312 310 311 309 307 306 308 304 305 303 301 302 300 299 298 296 295 294 293 292 291 290 289 288 297 287 286 284 285 283 281 282 279 278 280 277 275 276 274 272 271 273 269 270 267 266 265 268 263 262 264 261 259 258 260 257 256 255 254 252 253 250 249 251 248 246 247 245 244 242 241 240 239 243 238 237 235 236 234 233 232 231 230 229 227 228 225 226 223 224 221 222 220 219 217 218 215 214 213 212 216 210 211 208 209 206 205 207 203 202 204 200 199 201 197 198 196 195 194 192 193 190 191 188 187 186 189 184 183 185 181 182 180 179 177 178 175 174 173 176 172 171 170 168 169 166 167 164 163 165 161 160 162 159 158 157 156 155 153 154 151 150 149 148 147 152 145 146 143 142 144 141 139 140 137 136 138 134 135 132 133 130 131 129 128 127 126 125 124 122 123 120 121 118 117 119 116 114 115 112 111 113 110 108 109 106 105 107 103 102 101 104 99 98 100 97 96 95 94 92 91 93 90 89 87 88 85 86 84 83 82 81 79 80 78 77 76 74 73 75 72 71 70 68 69 67 66 64 63 65 61 62 60 59 58 57 56 55 54 52 51 50 49 48 53 46 45 47 44 43 41 40 42 39 37 36 35 34 38 33 32 30 31 29 28 27 26 24 25 22 23 20 21 19 18 17 15 16 14 12 11 10 9 8 13 7 5 6 4 2 3 1
100 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98
98 99 96 95 97 94 92 93 91 89 88 87 90 86 85 84 82 83 81 80 78 79 76 77 74 73 75 72 71 70 69 67 66 68 65 64 62 61 63 60 58 57 59 56 54 53 52 55 50 49 51 48 47 46 44 45 43 41 42 39 40 37 38 36 34 33 35 31 32 29 30 28 27 26 25 23 22 24 21 20 18 17 19 16 14 15 12 13 11 10 9 7 6 8 4 5 2 3 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
39 38 37 35 36 33 34 31 32 30 29 28 27 26 25 24 23 22 21 20 19 18 17 16 15 14 13 11 12 9 10 8 7 6 4 5 3 2 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
39 38 36 37 35 34 33 32 30 31 28 29 27 26 25 24 22 21 20 23 18 17 16 19 15 14 13 12 11 10 8 7 6 9 4 3 5 2 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
39 38 36 37 35 33 32 34 31 30 29 27 28 26 25 24 23 22 20 21 19 18 17 16 14 15 13 12 11 10 8 9 7 5 6 4 2 3 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
38 39 37 36 35 34 33 32 31 30 28 29 27 26 25 24 23 22 21 20 19 18 17 16 15 14 13 12 11 10 9 8 7 6 5 4 3 2 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
38 39 37 35 36 34 33 32 31 30 29 28 27 25 26 24 22 23 21 20 19 18 17 16 15 14 12 13 11 10 9 8 7 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 7 5 4 6 3 2 1
10 1
0 1 2 3 4 5 6 7 8
8 7 9 5 4 6 2 1 3
10 1
0 1 2 3 4 5 6 7 8
9 8 6 7 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 6 5 7 3 2 1 4
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 4 5 3 2 1
10 1
0 1 2 3 4 5 6 7 8
8 9 7 6 5 3 4 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 7 5 6 3 2 4 1
10 1
0 1 2 3 4 5 6 7 8
8 7 9 6 4 3 5 1 2
10 1
0 1 2 3 4 5 6 7 8
8 9 7 5 6 4 2 1 3
10 1
0 1 2 3 4 5 6 7 8
9 7 8 6 5 4 3 1 2
10 1
0 1 2 3 4 5 6 7 8
8 9 6 7 4 5 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 7 5 6 3 4 1 2
10 1
0 1 2 3 4 5 6 7 8
8 9 7 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 7 8 5 4 6 2 1 3
10 1
0 1 2 3 4 5 6 7 8
8 9 7 6 5 3 2 4 1
10 1
0 1 2 3 4 5 6 7 8
9 7 6 8 4 5 2 3 1
10 1
0 1 2 3 4 5 6 7 8
9 8 6 5 7 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 6 7 4 5 3 1 2
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
8 9 6 7 4 5 3 2 1
