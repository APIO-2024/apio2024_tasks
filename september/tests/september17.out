7ckgnn4wyi495puj3ibqf81dqvapyv6b
9
7
5
7
6
8
6
8
7
6
4
5
8
6
9
7
7
6
8
7
30
29
24
20
22
59
567
58
33
25
30
37
34
7
3
8
4
8
7
6
4
5
7
6
6
8
4
6
5
7
6
9
6
